org.envirocar.app/src/org/envirocar/app/views/login/SignupActivity.java

```diff
@@ -81,7 +81,7 @@
     private static final Logger LOG = Logger.getLogger(SignupActivity.class);
 
     private static final String EMAIL_REGEX = "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9-]+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
-    private static final String PASSWORD_REGEX = "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{6,}$";
+    private static final String PASSWORD_REGEX = "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=])(?=\\S+$).{6,}$";
```

Comment:
The current implementation is sufficent as of now. Also to implement this enhancement first the changes should be made on server so that the app gets supported with the new changes.

Reply:
Should I revert this change?